### DISCUSSION TITLE
Delete repos

### DISCUSSION THREAD

**Discussion by tillwenke** [Hitchwiki/hitchwiki#197] [Q&A]
I gave some repos a ["delete" topic tag](https://github.com/search?q=topic%3Adelete+org%3AHitchwiki&type=Repositories) where I was not sure what there purpose is. most of them were WIP but untouched for a while.
For the sake of providing a clean organization to newcomers it d be fine with me to delete those repos. If not one should provide a better description of their purpose. I also assume that the technology used in those repos is not what one would use today so there is really no need to keep them to integrate them at some point in the future.
@Hitchwiki/admins @simison

**Comment by guaka** [comment]
Yeah makes sense.

Let's say we will move forward with deletion on or after 2024-03-13 if no one raises an objection.

**Reply by tillwenke** [reply]
agree :)

**Reply by tillwenke** [reply]
@Hitchwiki/devs

**Comment by tillwenke** [comment]
Done
